tests/cypress/scripts/filters.js

```diff
@@ -1,116 +0,0 @@
-/** *****************************************************************************
- * Licensed Materials - Property of Red Hat, Inc.
- * Copyright (c) 2020 Red Hat, Inc.
- ****************************************************************************** */
-
-import { searchPage, searchBar } from '../views/search'
-
-export const simple = (filter) => {
```

Comment:
The functions within this file weren't actually returning anything. Therefore, the test has been running off of dead code; therefore, I removed this file and contained the filters within the test file.